README.md

```diff
@@ -25,9 +25,17 @@ In this challenge use `Test Driven Development` to build a RESTful API using Nod
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+
+  A - `describe()` is a way to encompass a suite of tests meanwhile `it()` is used to show what a specific test will test.
+
 1. What is the point of `Test Driven Development`? What do you think about this approach?
+
+  A- TDD is important because it for one makes sure we write clean code. It also makes sure that when we refactor our code, nothing breaks. It is a safety net. I personally really like TDD and I plan on using it in my professional career.
```

Comment:
Yup and it tries to minimize side effects by making our code predictable. Glad you like it 👍